src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+    
+    public final String value;
+    private boolean isPrivate;
+    
+    /**
+     * Validates given address.
```

Comment:
you are misleading the developers here; this constructor is not doing any validation, no?